Content/ContentImplementing.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ContentImplementing.swift
+//  RealifeTech
+//
+//  Created by Mickey Lee on 02/03/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import Foundation
+import RealifeTech_CoreSDK
+
+public class ContentImplementing: Content {
+
+    let dispatcher: GraphQLDispatching
+
+    public init(dispatcher: GraphQLDispatching) {
+        self.dispatcher = dispatcher
+    }
+
+    public func getWebPage(
+        forType type: WebPage.`Type`,
+        completion: @escaping (Result<WebPage, Error>) -> Void
+    ) {
+        makeGetWebPageQuery(by: type, completion: completion)
+    }
+
+    public func createWebPageView(
+        forType type: WebPage.`Type`,
+        webPgaeViewControllerDelegate: WebPageViewControllerDelegate?,
+        completion: @escaping (UIViewController) -> Void
+    ) {
+        makeGetWebPageQuery(by: type) { result in
+            let webPageViewController = WebPageViewController(result: result, delegate: webPgaeViewControllerDelegate)
+            completion(webPageViewController)
+        }
+    }
+
+    private func makeGetWebPageQuery(by type: WebPage.`Type`, completion: @escaping (Result<WebPage, Error>) -> Void) {
+        let pageType = ApolloWebPageType(rawValue: type.rawValue)
+        dispatcher.dispatch(
+            query: ApolloType.GetWebPageByTypeQuery(type: pageType),
+            cachePolicy: .returnCacheDataElseFetch) { result in
```

Comment:
having issues with this in frontier at the moment 
https://livestyled.atlassian.net/browse/ID-771

we may need to change the caching policy potentially based on the outcome

Reply:
This returns the local cache first and then returns the remote response just like the `concat` in the RemoteDiskCacheDataProviding. I think the best place to change the webView's cache policy is in the frontier.ios.